lib/ansible/module_utils/net_tools/nios/api.py

```diff
@@ -159,7 +159,7 @@ def member_normalize(member_spec):
                        'ha_port_setting', 'lan_port_setting', 'lan2_physical_setting',
                        'lan_ha_port_setting', 'mgmt_network_setting', 'v6_mgmt_network_setting']
     for key in member_spec.keys():
-        if key in member_elements:
+        if key in member_elements and not(member_spec[key] is None):
```

Comment:
@krisvasudevan I think you are trying to check if the `member_spec[key] is not None`, if yes plz modify the check as:
```suggestion
        if key in member_elements and member_spec[key] is not None:
```

Reply:
@justjais 
I've committed your changes. Thanks for taking the time out to review this.

Reply:
@krisvasudevan changes lgtm, once CI passes I'll merge the PR, also as suggested in mail plz create a backport PR for the changes to take effect in `2.8.1`